app/src/main/java/org/sagebionetworks/research/mpower/tracking/TrackingMenuFragment.java

```diff
@@ -0,0 +1,190 @@
+package org.sagebionetworks.research.mpower.tracking;
+
+import static com.google.common.base.Preconditions.checkState;
+
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.support.annotation.ColorRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.view.GestureDetectorCompat;
+import android.view.GestureDetector.SimpleOnGestureListener;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.sagebionetworks.research.mpower.R;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.BindViews;
+import butterknife.ButterKnife;
+import butterknife.Unbinder;
+
+public class TrackingMenuFragment extends Fragment {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TrackingMenuFragment.class);
+
+    private static final List<Integer> MEASURING_LABELS =
+            Arrays.asList(R.string.measuring_center_label, R.string.measuring_left_label, R.string.measuring_right_label);
+
+    private static final List<Integer> TRACKING_LABELS =
+            Arrays.asList(R.string.tracking_center_label, R.string.tracking_left_label, R.string.tracking_right_label);
+
+    // TODO for now icons have a white background, fix this when design gets the images without the background.
+    private static final List<Integer> MEASURING_ICONS =
+            Arrays.asList(R.drawable.walk_and_balance_icon, R.drawable.finger_tapping_complete_icon, R.drawable.tremor_icon);
+
+    private static final List<Integer> TRACKING_ICONS =
+            Arrays.asList(R.drawable.medication_icon, R.drawable.symptom_icon, R.drawable.trigger_icon);
+
+    // TODO get the gesture recognizer working.
+    private class FlingListener extends SimpleOnGestureListener {
+        @Override
+        public boolean onFling(MotionEvent event1, MotionEvent event2, float velocityX, float velocityY) {
+            if (event1.getY() < event2.getY()) {
+                // we have a down fling
+                setShowingContent(false);
+            } else {
+                setShowingContent(true);
+            }
+
+            return true;
+        }
+    }
+
+
+    private static final int ANIMATION_DURATION = 150;
+    @ColorRes
+    private static final int SELECTED_COLOR = R.color.royal500;
```

Comment:
What do you think about defining a theme to reflect SageDesign, the theme would have something like this for its attributes:
colorPrimary
colorPrimaryDark
colorPrimaryLight
colorSecondary

mPower would set its theme so 

colorPrimary = royal500
colorPrimaryDark = royal400
colorPrimaryLight = royal700
colorSecondary = butterscotch500

in XML we can reference color="?attr/colorPrimary"

I _think_ this should cover the main colors? 
There's more once we cover active/hover/pressed/disabled states and light/dark modes for buttons and links. And certain steps (like completion steps) might make sense to have a Torquoise theme

I took a brief stab at it in my current branch to try and fix some styling, but let's see if we can come up with a a good set of themes/styles now that we have stuff spread out across so many modules

Reply:
I think this should be done in a separate pr to avoid too many conflicts. Thoughts?

Reply:
Good idea